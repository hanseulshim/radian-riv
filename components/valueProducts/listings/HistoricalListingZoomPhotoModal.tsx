import React from 'react'
import Modal from 'components/common/Modal'

interface Props {
  closeModal: () => void
  photo: string
}

export default function HistoricalListingZoomPhotoModal({
  closeModal,
  photo
}: Props) {
  return (
    <Modal closeModal={closeModal} title={''} percent={75} id="zoom-photo" dark>
      <img className="photo" src={photo} alt="photo" />
    </Modal>
  )
}
