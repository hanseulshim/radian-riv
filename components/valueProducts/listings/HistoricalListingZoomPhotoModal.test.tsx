import * as React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import HistoricalListingZoomPhotoModal from './HistoricalListingZoomPhotoModal'

describe('HistoricalListingZoomPhotoModal', () => {
  let component
  beforeAll(() => {
    const props = {
      closeModal: () => {},
      photo: ''
    }
    component = shallow(<HistoricalListingZoomPhotoModal {...props} />)
  })
  it('should have img', () => {
    expect(component.find('.photo')).toHaveLength(1)
  })
  it('should match snapshot', () => {
    expect(toJson(component)).toMatchSnapshot()
  })
})
