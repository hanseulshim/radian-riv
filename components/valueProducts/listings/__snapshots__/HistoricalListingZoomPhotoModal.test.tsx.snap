// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoricalListingZoomPhotoModal should match snapshot 1`] = `
<Modal
  closeModal={[Function]}
  dark={true}
  id="zoom-photo"
  percent={75}
  title=""
>
  <img
    alt="photo"
    className="photo"
    src=""
  />
</Modal>
`;
